File: Dino and fetters.cpp
// https://basecamp.eolymp.com/en/problems/9640

#include <bits/stdc++.h>
using namespace std;
int main()
{
	vector<int> v;
	long n, s, a, sum = 0;
	cin >> n;
	for (int i = 0; i < 2 * n; i++)
	{
		cin >> s >> a;
		sum += s;
		v.push_back(a - s);
	}
	sort(v.rbegin(), v.rend());
	for (int i = 0; i < n; i++)
	{
		sum += v[i];
	}
	cout << sum;
}

File: Dino and fetters.py
# https://basecamp.eolymp.com/en/problems/9640

def main():
    n = int(input())  
    v = []  
    sum_val = 0  

    for _ in range(2 * n):
        s, a = map(int, input().split()) 
        sum_val += s 
        v.append(a - s) 

    
    v.sort(reverse=True)

    for i in range(n):
        sum_val += v[i]

    print(sum_val) 

if __name__ == "__main__":
    main()